### PR TITLE
Fix: Keeps displaying "Connecting"

### PR DESCRIPTION
**Pull request by hileix**
Follow: https://compass-tech.atlassian.net/browse/ADP-3074

### CODE REVIEW

File: src/shells/webextension/manifest.json [hunk 1]
```diff
@@ -34,7 +34,12 @@
     "persistent": false
   },
 
-  "permissions": ["storage"],
+  "permissions": [
+    "storage",
+    "file:///*",
```
Comment (godtail) [inline src/shells/webextension/manifest.json:39]: With adding this permission will cause a permission confirm modal when installing the plugin?

Comment (hileix) [reply src/shells/webextension/manifest.json:39]: @godtail 
It seems not.
The official plugin requires these permissions. I previously removed these permissions and found that without them, the issue described in ADP-3074 would occur.
So, I added those permissions back.

Comment (godtail) [reply src/shells/webextension/manifest.json:39]: Sounds good!

**Review by godtail — APPROVED** [review]

**Review by godtail — APPROVED** [review]
LGTM!

**Review by godtail — APPROVED** [review]

**Pull request merged**